File: docs/UprimerPay/acquire_sdk_python_v1.0/AcquireSdk/request/queryRequest.py
import sys
sys.path.append('../') 
from core.request import Request
import json

# 
#   交易查询,可以查询交易和退款
#   文档地址：https://uprimer.net/open/docs/api#/api_jycx
#  
class QueryRequest(Request):
    def __init__(self):
        super().__init__()

    def create(self,originalId):              
        url = Request.HOST + f"/api/acquire/payment/{originalId}/get"           
        response = self.httpGet(url=url)
        return json.loads(response.text) 

if __name__ == '__main__': 

    originalId = "1020240726000028" 
    request = QueryRequest()
    response = request.create(originalId)
    print()
    print()
    print(f"接口返回={response}")

File: docs/UprimerPay/acquire_sdk_java_v1.0/AcquireSdk/src/main/java/com/acquire/sdk/request/QueryRequest.java
package com.acquire.sdk.request;

import com.acquire.sdk.core.Request;
import com.acquire.sdk.enums.UriEnum;

/**
 * 交易查询,可以查询交易和退款
 * 文档地址：https://hwonline.cleattle.com/doc/acquire.html#id7
 */
class QueryRequest extends Request
{
    public static void main(String[] args) 
    {
        try {

            QueryRequest request = new QueryRequest();
            String originalId = "1020240726000028";
            String url = HOST + UriEnum.URI_TRANSACTION_QUEURY.getUri();    
            url = url.replace("{originalId}", originalId); 
            String response = request.httpGet(url);
            System.out.println();  
            System.out.println("QueryRequest: ");   
            System.out.println();   
            System.out.println("response: " + response);
            System.out.println();
        } catch (Exception e) {
            e.printStackTrace();
        }
       
    }
   
}

